src/config/jiraConfig.ts

```diff
@@ -0,0 +1,3 @@
+export const JIRA_BOARD_URL = "https://govukverify.atlassian.net/rest/api/3/issue";
```

Comment:
SSM Params ?

Reply:
What's the advantage of SSM params here over hard coded?

These don't vary be environment, which would be our usual benefit.
They're not secret.
And to change them in config would mean having to commit code, merge and release code elsewhere, so there's a similar go-to-live cadence.

To me, makes more sense to have these next to the code that uses them, with good commits and a version control history. Especially as to run them locally these are values that turn up in .env and someone might want to snoop the integration to work out how.

I figure the lower complexity option is good value here?

Reply:
Changing SSM param does not require code change or commit. In our case it is updating value manually in AWS console and restarting pipeline.

Reply:
Do we not populate non-secret SSM params via Infrastructure as Code?
If not I would have thought the version control history is the real asset here?

We should try and avoid clickops where we can?